src/api/folderResourceMetaApi.ts

```diff
@@ -101,6 +112,9 @@ export const fetchFolderResourceMeta = async (
   } else if (resource.resourceType === "multidisciplinary") {
     const res = await fetchAndTransformResourceMeta([resource], context, "multidisciplinary");
     return res[0] ?? null;
+  } else if (resource.resourceType === "topic") {
+    const res = await fetchTopicMeta([resource], context, "topic");
```

Comment:
Kan ikkje denne bruke fetchAndTransformResourceMeta? Den brukes jo for multidisciplinary som er emner det også

Reply:
Ja, man kan, men litt usikker på om jeg vil. `fetchAndTransformResourceMeta` bruker artikkel-ID, fordi det er artikkel som er i fokus og hentes. Node utledes fra artikkel-ID'en. `fetchTopicMeta` gjør det motsatte, og henter artikkel basert på noden man ber om. Separerte de egentlig for å vise at de har hvert sitt formål, men kan sikkert slå de sammen hvis du mener det er bedre!